extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -13,3 +13,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+amp-sidebar {
+  color: black;
+  background-color: #efefef;
+  height: 100vh;
+  --story-page-vh: 1vh;
+  top: 0;
+  max-height: 100vh !important;
+  max-width: 80vw !important;
+  min-width: 45px !important;
```

Comment:
Keeping `!important` for these three properties.  This means the user will also need to include `!important` to overwrite.  I think this is fine since this is how it was done in 0.1, but open to opinions.

Reply:
We should just see `0.2` version for where `!important` is used and where it's not.

Reply:
`0.1` and `0.2` have identical CSS.  Which includes `!important` for `max-height` and `min-width` (out of the configurable properties).  So I have also added `!important` for `max-width` which I think makes sense since all three of these properties are closely related. :)

Reply:
Thought about this some more.  I think we should drop `!important` on `max-width`. There's one important case where a sidebar can self-size on width, and it's desirable to override the `max-width`. This reasoning is a good idea to comment in the CSS file.

Reply:
Thanks makes sense, I have added the following comment in the code: 

/** 
   * Max width allowed to be configurable by user.  If user wants width to be
   * controlled by content size and max-width, then this property needs to be
   * configurable.
   */